src/display/video/VideoHTML5.js

```diff
@@ -1765,9 +1804,9 @@ FORGE.VideoHTML5.prototype._onEventHandler = function(event)
             break;
 
         case "volumechange":
-            // I do not dispatch the volume change if readyState is 0. Because
-            // I set the volume at 0 when I create the video element, it is
-            // not usefull to dispatch this internal volume change ?
+            //I do not dispatch the volume change if readyState is 0. Because
+            //I set the volume at 0 when I create the video element, it is
+            //not usefull to dispatch this internal volume change ?
             if (this._onVolumeChange !== null && element.readyState !== 0)
```

Comment:
While you're at it, can you change this to `element.readyState !== HTMLMediaElement.HAVE_NOTHING` ? Same above with `durationchange` and `HTMLMediaElement.HAVE_METADATA`.